### PR TITLE
Feature/signup

### PR DESCRIPTION
**Pull request by teresafranxman97**
"As a user, I should be presented with the option to apply to the mentor or mentee program". Initially, the signup page directed users to the mentee application form and gave no option to apply to become a mentor. This feature presents users with both options so that they are able to navigate to the correct form whether they want to become a mentee or a mentor.  

*** Loom video link: https://www.loom.com/share/7c91f4963edd45d5aa7144d05303ad98 ***

## Type of change
- Updated signup.js component
- Created cards that presented a user with the option to sign up for the mentor or mentee program 
- directed users to the correct application form 
- created additional signup.css file to make the cards follow the wireframe that was created for this component

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by ayv8er — APPROVED** [review]
This is exactly what the Stakeholder was talking about! This clears the way for registration and intake to take place in one form

**Comment by RaphaelHebert** [comment]
Good job, the code is clear and easy to read. Good use of antd.

**Review by cyberkade — APPROVED** [review]
This turned out really well, I'm loving it! Great loom video as well, very informative.

**Review by nhamilton1 — APPROVED** [review]
Fantastic work and a really helpful loom video! I would be interested in seeing some media query breakpoints if you decide to implement them.

File: src/components/pages/RoleSignup/Styles/signup.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+.title {
+    font-size: 20px;
+    font-weight: bold;
+}
+
+.divider {
+    margin: 4em;
+}
```
Comment (rickmansfield) [inline src/components/pages/RoleSignup/Styles/signup.css:24]: There should be a new line at the end of each component block. I hope Ryan sees that I'm being nit-picky. LOL. Wink

**Review by rickmansfield — APPROVED** [review]
Really like this result both aesthetically and functionally

**Review by Retrofitt — APPROVED** [review]
Code is clean and easy to read, Loom video does a good job of walking us through your code, and the page itself looks nice. Great work!

**Pull request merged**